etl/steps/data/garden/papers/2022-11-03/zijdeman_et_al_2015/__init__.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore_table
+from owid.datautils import geo
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+COUNTRY_MAPPING_PATH = N.directory / "countries.json"
+EXCLUDED_COUNTRIES_PATH = N.directory / "excluded_countries.json"
+METADATA_PATH = N.directory / "meta.yml"
+
+
+def run(dest_dir: str) -> None:
+    log.info("zijdeman_et_al_2015.start")
+
+    # read dataset from meadow
+    ds_meadow = Dataset(DATA_DIR / "meadow/papers/2022-11-03/zijdeman_et_al_2015")
+    tb_meadow = ds_meadow["zijdeman_et_al_2015"]
+
+    # Create table
+    tb_garden = make_table(tb_meadow)
+
+    # Initiate dataset
+    ds_garden = Dataset.create_empty(dest_dir)
+    ds_garden.metadata = ds_meadow.metadata
+    ds_garden.metadata.update_from_yaml(METADATA_PATH)
+
+    # Add table
+    ds_garden.add(tb_garden)
+    ds_garden.save()
+
+    log.info("zijdeman_et_al_2015.end")
+
+
+def make_table(tb_meadow: Table) -> Table:
+    df = pd.DataFrame(tb_meadow)
+
+    # Create table
+    tb_garden = underscore_table(Table(df))
+    tb_garden.metadata = tb_meadow.metadata
+    for col in tb_garden.columns:
+        tb_garden[col].metadata = tb_meadow[col].metadata
+
+    # Clean countries
+    tb_garden = format_columns(tb_garden)
+    tb_garden = clean_countries(tb_garden)
+
+    # Set index
+    tb_garden = tb_garden.set_index(["country", "year"])
```

Comment:
Consider using `set_index(["country", "year"], verify_integrity=True)`, and optionally followed by `.sort_index()` just to make it extra neat.